09/Time.java

```diff
@@ -45,4 +45,18 @@ public static Time Random() {
         int randomMinutes = rand.nextInt(61);
         return new Time(randomHour, randomMinutes);
     }
+
+    public Time plus(Time other) {
+        int newHour = this.hours + other.hours;
+        int newMinuets = this.minutes + other.minutes;
+        if (newMinuets <= 60) {
```

Comment:
```suggestion
        if (newMinuets >= 60) {
```
Logic is flipped here too - see my other comment.